nf_core/lint.py

```diff
@@ -112,11 +112,11 @@ class PipelineLint(object):
             params.help = false
             params.outdir = './results'
             params.bam = false
-            params.singleEnd = false
+            params.single_end = false
             params.seqtype = 'dna'
             params.solver = 'glpk'
             params.igenomes_base = './iGenomes'
-            params.clusterOptions = false
+            params.cluster_options = false
```

Comment:
this is probably just confusion at my side, but what exactly is `params.clusterOptions` used for?

Just asking because there is also `process.clusterOptions` which would need to stay camelCase because it's a process directive

Reply:
Ah good point, I will revert. It just makes it easier to supply additional options to the cluster, typically in [institutional configs](https://github.com/nf-core/configs/blob/8e5cc7838c93b1dacdff6bb87e388fab709735f0/conf/uppmax.config#L14).